### ISSUE TITLE
flutter web ReactiveBuilder:  Unsupported operation: Platform._operatingSystem

### ISSUE THREAD

**Issue by 4F2E4A2E** [escamoteur/rx_widgets#19]
Hello there and thank you for rx_widgets!

On my journey in finding a reactive way of communication and reactive ui in flutter [5], the road led me here, to you guys.

Everything is working fine on iOS, but I just want to report this exception [2] being thrown for this code [1] on web, using latest stable version of flutter [3]:

1: https://github.com/4F2E4A2E/getx-websocket-example/blob/1.0.1/lib/view/detail_v2.dart#L17
```dart
...
ReactiveBuilder<UserDataProfile>(
    stream: controller.userDataProfile.stream,
    builder: (BuildContext context, data) {
        return Text(data.name);
    },
),
...
```

2: Unsupported operation: Platform._operatingSystem

```bash
======== Exception caught by widgets library =======================================================
The following UnsupportedError was thrown building StreamBuilder<UserDataProfile>(dirty, state: _StreamBuilderBaseState<UserDataProfile, AsyncSnapshot<UserDataProfile>>#48108):
Unsupported operation: Platform._operatingSystem

The relevant error-causing widget was: 
  ReactiveBuilder<UserDataProfile> getx-websocket-example/lib/view/detail_v2.dart:30:15
When the exception was thrown, this was the stack: 
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 236:49     throw_
dart-sdk/lib/_internal/js_dev_runtime/patch/io_patch.dart 244:5                  _operatingSystem
dart-sdk/lib/io/platform_impl.dart 56:40                                         get operatingSystem
dart-sdk/lib/io/platform.dart 64:45                                              get _operatingSystem
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 825:8  get
...
====================================================================================================

======== Exception caught by widgets library =======================================================
Unsupported operation: Platform._operatingSystem
The relevant error-causing widget was: 
  ReactiveBuilder<UserDataProfile> getx-websocket-example/lib/view/detail_v2.dart:17:18
```

3: flutter version
```bash
$ flutter doctor       
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.0.4, on macOS 11.2.3 20D91 darwin-x64, locale en-AT)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2019.3.3)
[✓] VS Code (version 1.55.0)
[✓] Connected device (2 available)

• No issues found!
Time: 0h:00m:08s  
```

4: flutter pub outdated
```bash
transitive dependencies:
rxdart           *0.25.0  *0.25.0     *0.25.0     0.26.0   
```

5: https://stackoverflow.com/questions/66933598/flutter-getx-first-null-value

P.s.: Please also notice [4]

**Comment by 4F2E4A2E** [comment]
The best way I could find on how to realize this, was a mix of getx and rx_widgets.
https://stackoverflow.com/a/66975725/543426

**Comment by escamoteur** [comment]
Hi,
sorry for the delay. should be fixed in V4.0.1 attention that is the now null safe version
